views/partials/dashboardResultsEApp.ejs

```diff
@@ -0,0 +1,146 @@
+<div class="container">
+    <form id="dashboard-search-table-form" method="post" action="/dashboard">
+        <label for="dashboard-search-filter" class="visuallyhidden">Search by reference or service</label>
+        <input type="text" name="dashboardFilter" id="dashboard-search-filter" style="height: 40px;padding: 0 5px;"
+            placeholder="Search by reference or service" title="Search" value="<%= searchCriteria %>"
+            class="govuk-input govuk-!-width-one-third">
+        <input class="submit" type="submit" value="Search">
+        &nbsp;
+        <% if (searchCriteria) { %><a href="/dashboard" id="dashboard-clear-results">Clear search filter</a>
+            <%}%>
+    </form>
+</div>
+<table id="previousApplications">
+    <caption class="visually-hidden">Application <% if (searchCriteria) { %>search results<%}else{%>history<%}%> table.
+                    Can be sorted by clicking the table headers.</caption>
```

Comment:
I suppose only people with screenreaders would be able to "see" (hear) this. Those users usually aren't "clicking". maybe selecting is a better term?

Reply:
I think so, this is something I took from the dashboardResults.ejs page but yeah it does make sense.